setup.cfg

```diff
@@ -21,6 +21,8 @@ classifiers =
     Programming Language :: Python :: 3.7
```

Comment:
I wonder whether this will remain true... ;-)

Reply:
We're still testing `3.7` so this should be fine, but I've dropped the `3.6` version since that one has been EOLed for a while anyway. (I believe 3.7 was still receiving security releases up until couple months go...)